### PR TITLE
implement php into html, search toggle

### PR DESCRIPTION
**Pull request by faithyeji**
Implemented php, established connection to primary database/server, and added a working search button to toggle on click and pull results.

### CODE REVIEW

File: src/components/Header/header.css
Comment (roryvision) [inline src/components/Header/header.css:1]: The search box is not vertically aligned to the rest of the elements in the header.

File: src/pages/dashboard.php [hunk 1]
```diff
@@ -33,5 +39,36 @@
         <span>Sort</span>
       </button>
     </div>
+
+<!--    populating projects-->
```
Comment (roryvision) [inline src/pages/dashboard.php:43]: Projects should be in the container.

**Review by roryvision — COMMENTED** [review]
Connection should only be open when used, and must be closed after. TODO: Do the HTML/CSS for search and result pages. Implement people cards.

File: src/components/Header/header.css
Comment (faithyeji) [reply src/components/Header/header.css:1]: Aware of this—search was a last minute deadline change but we will run through and polish out css tonight or tomorrow.

**Review by roryvision — APPROVED** [review]

**Pull request merged**